add Tesla Wall Connector plugin

kenyon opened the pull request:
This plugin graphs some data accessible via the Tesla Wall Connector's HTTP API.

Example here: https://beta.kenyonralph.com/munin/tesla-day.html

sumpfralle commented:
Thank you for contributing this new plugin!

Could you please turn the plugin documentation into the `perldoc` format in order to beautify the plugin gallery representation (and for the manpage)?

Maybe you also want to [add example graphs](http://guide.munin-monitoring.org/en/latest/develop/plugins/plugin-gallery.html#rules-for-plugin-contributors)?

Regarding the plugin name: is `wall_connector` Tesla's official name for this device? (I do not know their branding style, thus I would trust you to pick a long-term usable name)

Thank you!

kenyon commented:
Thanks for the review. Converted the documentation to `perldoc`. Added example graph images. Yes, the device is called the Wall Connector: https://shop.tesla.com/product/wall-connector

sumpfralle commented:
Great - thank you for contributing this new plugin!
